File: 151001/Novik/DC/src/Publisher/Models/DTOs/Responses/UserResponseTo.cs
namespace Publisher.Models.DTOs.Responses;

public class UserResponseTo
{
    public long id { get; set; }
    
    public String login { get; set; } = String.Empty;

    public String password { get; set; } = String.Empty;
    
    public String firstname { get; set; } = String.Empty;

    public String lastname { get; set; } = String.Empty;
}

File: 151001/Matsiushenko/lab2/src/main/java/by/bsuir/vladislavmatsiushenko/dto/UserResponseTo.java
package by.bsuir.vladislavmatsiushenko.dto;

import jakarta.validation.constraints.Size;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UserResponseTo {
    int id;
    @Size(min = 2, max = 64)
    String login;
    String password;
    String firstname;
    String lastname;
}
